app/client/components/svgs/profileIcon.tsx

```diff
@@ -3,14 +3,21 @@ import { palette } from "@guardian/src-foundations";
 import React from "react";
 
 interface ProfileIconProps {
-  css?: SerializedStyles;
+  additionalCss?: SerializedStyles;
```

Comment:
is this actually being used? (it looks like the old `css?` prop wasn't used)

Reply:
It's used in dropdownNav for the little profile roundel next the the title "My Account" in order to position it within the circle